### PR TITLE
fix #8360 feat(nimbus): Change root folder app to experimenter

### LINKED ISSUE #8360: Rename app root folder to Experimenter
Rename `app` root folder to `experimenter`
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3199)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- We will be adding cirrus in experimenter

This commit

- Rename `app` to `experimenter` and make necessary changes

### CODE REVIEW

File: experimenter/experimenter/settings.py [hunk 1]
```diff
@@ -60,7 +58,7 @@
 ALLOWED_HOSTS = [HOSTNAME]
 
 if DEBUG:
-    ALLOWED_HOSTS += ["localhost", "nginx"]  # pragma: no cover
+    ALLOWED_HOSTS += ["localhost", "nginx", "0.0.0.0"]  # pragma: no cover
```
Comment (yashikakhurana) [inline experimenter/experimenter/settings.py:61]: @jaredlockhart after changing app to experimenter, I am not sure what I am missing but localhost is not working, I have to explicitly add 0.0.0.0, do you know why?

**Comment by yashikakhurana** [comment]
@jaredlockhart @jrbenny35 @jeddai @eliserichards it is ready for a review, thank you 🙏

**Review by jeddai — DISMISSED** [review]
I've smoke tested the main app after making the change below – had to run `docker-compose build nginx` to get it to actually rebuild the image. 

Overall looks like it works well other than the nginx issue!

File: docker-compose.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 version: "3"
 
 services:
-  app:
-    image: app:dev
+  experimenter:
```
[the remark this answers is not in the document]
Comment (yashikakhurana) [reply docker-compose.yml:4]: thanks @jeddai ❤️ , all done now

**Review by jeddai — APPROVED** [review]
Looking great! I've smoke tested it and it seems like it's functioning as we'd expect!

**Review by jrbenny35 — APPROVED** [review]

**Review by eliserichards — APPROVED** [review]
Working for me! 🚢 🚀

**Comment by yashikakhurana** [comment]
changed make commands to rebuild the image in this [commit](https://github.com/mozilla/experimenter/pull/8375/commits/7a01727c128607f2de2486fca6f4752f3e978c1b), asking for re-review from all, @jeddai @jrbenny35 @eliserichards @jaredlockhart 🙏

**Review by jaredlockhart — APPROVED** [review]
Okay.  Wow.  Great job @yashikakhurana .  This is a huge undertaking but testing locally everything seems tip top.  Thank you.  This is an exciting step as we expand from just Experimenter to the entire Nimbus infrastructure.  Now let's land this thing and go home!

ulXfUdG

**Review by mikewilli — APPROVED** [review]

**Review by eliserichards — APPROVED** [review]
💯

**Comment by yashikakhurana** [comment]
I am rebasing and super scared to merge this 😨

**Pull request merged**